sympy/utilities/runtests.py

```diff
@@ -210,15 +211,16 @@ def run_in_subprocess_with_hash_randomization(
 
     # First check if the Python version supports hash randomization
     # If it doesn't have this support, it won't reconize the -R flag
+    current_dir = dirname(dirname(os.path.dirname(__file__)))
```

Comment:
I think it would be better to get the directory from the sympy package. This code assumes that it is in this particular file in this particular place in the SymPy code tree but it could get moved.

Something like:
```
import sympy
sympy_root = dirname(dirname(sympy.__file__))
```